### DISCUSSION TITLE
Set step size to y-axis and adjust min max values

### DISCUSSION THREAD

**Discussion by siroaleksi** [patrykandpatrick/vico#496] [Questions] [answered]
Is it possible to define a step size for the y axis? E.g. steps of 10. Also is it possible to define the max and min values for the y-axis? Again e.g. the max value is dataset's max value + 10 and minimum value - 10.  Image above is the current solution and I would like to achieve the lower image.

![Screenshot_20231215-140511__01](https://github.com/patrykandpatrick/vico/assets/97961365/a9b3b594-c3e0-4591-8460-4f6e086ed63f)
![Screenshot_20231215-140511__01_this_I_want](https://github.com/patrykandpatrick/vico/assets/97961365/5802addf-d0c2-477b-b985-393a7c7b44db)

**Comment by patrickmichalik** [comment, marked as answer]
Hello! To modify the _y_ range, you can use `AxisValuesOverrider` (`AxisValueOverrider` in Vico 2). The `columnChart` and `lineChart` (`rememberColumnCartesianLayer` and `rememberLineCartesianLayer` in Vico 2) composable functions have `axisValuesOverrider` (`axisValueOverrider` in Vico 2) parameters. `ColumnChart` and `LineChart` (`ColumnCartesianLayer` and `LineCartesianLayer` in Vico 2) have properties with the same name. As for the label spacing, please see the `maxItemCount` parameter of `AxisItemPlacer.Vertical.default`. For greater flexibility, [Vico 2.0.0 Alpha 4](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.4) made this a lambda.

**Reply by siroaleksi** [reply]
Thank you for the answer! I'll check this out.

**Reply by siroaleksi** [reply]
This worked like I wanted. Thank you once again!

**Reply by patrickmichalik** [reply]
I’m glad to hear that. You’re welcome!

**Comment by patrickmichalik** [comment]
[Vico 2.0.0 Alpha 9](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.9) simplifies this by introducing `AxisItemPlacer.Vertical.step`, which lets you directly specify the _y_ step for a `VerticalAxis`. Cheers!

**Reply by siroaleksi** [reply]
Thanks!
